ioctl/newcmd/config/config.go

```diff
@@ -128,6 +131,76 @@ func (c *info) reset() error {
 	return nil
 }
 
+// set sets config variable
+func (c *info) set(args []string, insecure bool) (string, error) {
+	switch args[0] {
+	case "endpoint":
+		if !isValidEndpoint(args[1]) {
+			return "", errors.Errorf("endpoint %s is not valid", args[1])
+		}
+		c.readConfig.Endpoint = args[1]
+		c.readConfig.SecureConnect = !insecure
+	case "analyserEndpoint":
+		c.readConfig.AnalyserEndpoint = args[1]
+	case "wallet":
+		c.readConfig.Wallet = args[1]
+	case "explorer":
+		lowArg := strings.ToLower(args[1])
+		switch {
+		case isValidExplorer(lowArg):
+			c.readConfig.Explorer = lowArg
+		case args[1] == "custom":
```

Comment:
`"custom"` can add description in `_configSetUseCmdLong`

Reply:
Added that but the Chinese translation may need tweaking 😬

Reply:
good.